### ISSUE TITLE
cam multiplied by GT label?

### ISSUE THREAD

**Issue by periakiva** [YudeWang/SEAM#27]
Both during training and inference the cam output is multiplied by the ground truth label.

training:
```
Line 123: cam_rv1 = F.interpolate(visualization.max_norm(cam_rv1),scale_factor=scale_factor,mode='bilinear',align_corners=True)*label
Line 129: cam_rv2 = visualization.max_norm(cam_rv2)*label
```
inference:
`Line 63: cam = cam.cpu().numpy() * label.clone().view(20, 1, 1).numpy()`

Is that done in error? How can we assume that labels are available during inference?

**Comment by YudeWang** [comment]
Hi @periakiva ,
Duplicate with issue #16 and I will close this. Feel free to reopen if any more question.
